Doc/library/crypt.rst

```diff
@@ -41,17 +41,24 @@ are available on all platforms):
 .. data:: METHOD_SHA512
 
    A Modular Crypt Format method with 16 character salt and 86 character
-   hash.  This is the strongest method.
+   hash based on the SHA-512 hash function.  This is the strongest method.
```

Comment:
Is it still the strongest now?

Reply:
I don't know what is stronger, SHA or Blowfish. But since SHA methods were added in glibc when the Blowfish method already was provided by the third-party library, I suppose that at least SHA-512 is not weaker (and likely is stronger) than Blowfish.

Reply:
You're right. Reference for the SHA2 methods is at https://www.akkadia.org/drepper/SHA-crypt.txt